### DISCUSSION TITLE
How to Select the Address for the Inscription

### DISCUSSION THREAD

**Discussion by abdullahe** [ordinals/ord#3534] [Q&A] [answered]
I created a few addresses with `ord wallet receive`. I can list them using this: `bitcoin-cli listreceivedbyaddress 0 true`
Is there a way to select the address with inscribe command? 
`ord wallet inscribe [OPTIONS] --fee-rate <FEE_RATE> --file <FILE>` this use doesn't mention how it can be done. 

Thanks for the help.

**Comment by raphjaph** [comment, marked as answer]
`ord wallet inscribe --destination <ADDRESS>`

**Reply by Peytony440** [reply]
And how to airdrop rune tokens to several thousand addresses? A batch? Do you have the code?
